doc/content/gateways/lora-basics-station/troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Troubleshooting LBS"
+description: ""
+---
+
+This section contains help for common problems you may encounter when using {{% lbs %}} to connect to {{% tts %}}.
+
+<!--more-->
+
+## What is my server address?
+
+This is the address you use to access {{% tts %}}. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
+
+CUPS uses the URI: `https://<server-address>:443`
+
+LNS uses the URI: `wss://<server-address>:8887`
+
+## How do I find the CA Trust?
+
+Some device manufacturers include [common CA's](https://www.ccadb.org/) in device firmware, and these devices should connect automatically to {{% tts %}}.
```

Comment:
This would be `CAs` (without the apostrophe). Please also update the other occurrences.